pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -30,16 +32,37 @@
     FULL, PARTIAL, NONE
   }
 
+  public enum Strategy {
+    OVERWRITE, INCREMENT
```

Comment:
add a TODO for custom

Reply:
gotcha resolved in https://github.com/apache/incubator-pinot/pull/6899/commits/61356a46850d3c0eeb9313845ec77d10769486ba